view/theme/quattro/dark/style.css

```diff
@@ -64,7 +64,7 @@
 .icon.unlock {
   background-image: url("../../../images/icons/22/unlock.png");
 }
-.icon.addon {
+.icon.plugin {
```

Comment:
This is not right, we changed all mentions of `plugin` to `addon`.

Reply:
Obviously not all. The files where just regenerated from the less files. But I'll have a look later.

Reply:
There are several files that contain "plugins", mostly templates and style files it seems. TBH I would rather fix this in a later separate PR

Reply:
Reverting this has fixed a bug I noticed before as well ;-) Will check the plugin/addon stuff hopefully tomorrow!